index.html

```diff
@@ -60,43 +71,51 @@ <h3>LET'S CONNECT</h3>
       </section>
     </div>
     <!-- ProjectsSection -->
-    <section class="projectsContainer">
+    <section class="projectsContainer" id="projects">
         <div class="project1">
           <div class="projImg">
-            <img src="./Images/mobileprojects/project1Img.svg" alt="project1img" />
+            <img class="mobileimg" src="./Images/mobileprojects/project1Img.svg" alt="project1img" />
+            <img class="desktop1img" src="./Images/desktop/image1.svg" alt="">
+          </div>
+          <div class="proj1text">
+
+            <h3>Tonic</h3>
+            <ul class="a">
+              <li>CANOPY</li>
+              <li>
+                <img src="./Images/icons/CounterDot.svg" alt="counterimg" />
+              </li>
+              <li>Back EndDev</li>
+              <li>
+                <img src="./Images/icons/CounterDot.svg" alt="counterimg" />
+              </li>
+              <li>2015</li>
+            </ul>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="b">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>
+                See project
+            </button>
```

Comment:
- [ ] [OPTIONAL] Just like the previous reviewer said [here](https://github.com/mj-e-boc/MJay-Portfolio/pull/4#discussion_r1159855976) Kindly fix the button interaction. 😃
https://github.com/mj-e-boc/MJay-Portfolio/pull/4#discussion_r1159855976